MotionMark/resources/runner/benchmark-runner.js

```diff
@@ -104,37 +104,40 @@ class BenchmarkRunner {
 
     _runBenchmarkAndRecordResults(state)
     {
-        var promise = new SimplePromise;
-        var suite = state.currentSuite();
-        var test = state.currentTest();
+        const suite = state.currentSuite();
+        const test = state.currentTest();
 
         if (this._client && this._client.willRunTest)
             this._client.willRunTest(suite, test);
 
-        var contentWindow = this._frame.contentWindow;
+        const contentWindow = this._frame.contentWindow;
         var self = this;
 
-        var options = { complexity: test.complexity };
+        const options = { complexity: test.complexity };
         Utilities.extendObject(options, this._client.options);
```

Comment:
As far as I can tell extendObject can be replaced entirely with the spread operator when building the options object here, like so:
```
const options = {
    complexity: test.complexity,
    ...this._client.options,
    ...Utilities.parseParameters(contentWindow.location),
}
```

Although this particular type of change is probably out of scope for this PR's purpose.

Reply:
`Utilities.extendObject` needs to be cleaned up separately